src/main/java/teammates/ui/webapi/endpoints/CronJobEndpoints.java

```diff
@@ -0,0 +1,31 @@
+package teammates.ui.webapi.endpoints;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import teammates.common.util.Const.CronJobURIs;
+
+/**
+ * API endpoints for cron jobs.
+ */
+public enum CronJobEndpoints {
```

Comment:
The cron job's URL should not be exposed right?

Reply:
I created the endpoints because I wasn't too sure if we might need it, but since it's unnecessary I will remove it.

Reply:
Both should not be exposed if you take a careful look at what they do.

Reply:
Yup the cron's URLs are used only in the backend so I understand why we don't need it. I'm not too sure about  the legacy URLs, but I suppose since it's legacy we're trying to avoid using them. 

I have removed both from my PR. Is there anything else I need to add/change? Otherwise I think this PR is ready. (There are more instructor modules with `HttpRequestService`, but this PR might become very large if I refactored them too. I'm planning to do them in another PR)